### ISSUE TITLE
javax.websocket

### ISSUE THREAD

**Issue by cdietrich** [eclipse/lsp4j#647]
build uses javax.websocket from orbit, which is no longer available in newer orbit versions.
we might have to use a newer orbit version for a newer gson eventually.

=> how to resolve? is jakarta.websocket a feasible alternative?

**Comment by pisv** [comment]
> => how to resolve? is jakarta.websocket a feasible alternative? so that we can drop the javax one?

I don't think so, see #536. Could not we just stick to the last available javax.websocket from Orbit?

**Comment by cdietrich** [comment]
I think we can, just makes the target file more complicated / less straight forward

**Comment by jonahgraham** [comment]
Note that this is a subissue of #874 to remove all deprecated API for 1.0 release. PR is ready to review (along with rest of deprecated API removal) in #881
